script/worker

```diff
@@ -130,12 +129,7 @@ $options{instance} ||= 1;
 my $worker = OpenQA::Worker->new(\%options);
 $worker->log_setup_info();
 
-sub catch_exit {
-    my ($sig) = @_;
-
-    log_info("Quit due to signal $sig");
-    $worker->stop(WORKER_COMMAND_QUIT);
-}
+sub catch_exit { $worker->handle_signal(@_) }
```

Comment:
this seems to be the only line that is not covered by tests which is what we expected, right? Can you add an `# uncoverable statement` here?